### ISSUE TITLE
Any demo app which we can see?

### ISSUE THREAD

**Issue by siddharthpal** [ErinCoughlan/create-ng1-wrapper#2]

**Comment by siddharthpal** [comment]
@ErinCoughlan I have created a repository which implements this wrapper library. It seems to work perfectly but in the console, I see an injector error. Can you please help in resolving the issue.
To run the app we need to do:

1. npm i && cd elements && npm i
2.  cd.. 
3.  npm run build:elements
4. cd elements
5. npm run start

[Demo repository](https://github.com/siddharthpal/ng1-custom-elements-wrapper)

**Comment by siddharthpal** [comment]
Live demo hosted on firebase [https://create-ng1-wrapper-app.firebaseapp.com/](url)

**Comment by ErinCoughlan** [comment]
Thank you! I tried to build the project yesterday, but didn't see the injector error you mentioned.

I assume that you were able to resolve it?


I am also working on a demo based on TodoMVC, but updating the AngularJS code took longer than expected (it was on AngularJS 1.3 and didn't have components or bindToController.)

**Comment by siddharthpal** [comment]
Yes, @ErinCoughlan. This issue happens only when we do production build. As of now, the app is not hosted with minified files so it is working. But now I see the PR for the *ngInject* build issue is already merged.
Now I will update the app with production build.

Thanks

**Comment by siddharthpal** [comment]
I tried to take a pull of the latest version but I don't see the changes. Can you please publish it?

**Comment by ErinCoughlan** [comment]
New version is now published!
